{"name": "Негативная проверка доверенного номера с символом '%'", "status": "passed", "description": "Проверка неудачного поиска клиента по доверенному номеру с символом '%'", "steps": [{"name": "Проверка сообщения об ошибке и кода ответа", "status": "passed", "start": 1715658647244, "stop": 1715658647244}], "parameters": [{"name": "proofNum", "value": "'996553565311%'"}], "start": 1715658647112, "stop": 1715658647245, "uuid": "872ba5fe-6d0b-48a4-adab-984772bda678", "historyId": "8491f76211981660967405d3cd41fefe", "testCaseId": "a6be3475c37a4f6fbc7e20e79cd9ff1a", "fullName": "207_service_test.TestProofNumSuite#test_get_info_client_has_special_symbol_in_proofNum_get_error_response", "labels": [{"name": "suite", "value": "Получение информации по клиенту (207 сервис)"}, {"name": "subSuite", "value": "Проверки по доверенному номеру"}, {"name": "host", "value": "PC-411-140322D"}, {"name": "thread", "value": "31860-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "207_service_test"}]}